docker-compose.dev.yml

```diff
@@ -21,11 +21,13 @@ services:
   spot:
     image: samply/rustyspot:main
     ports:
-      - 8080:8080
+      - 8055:8055
     environment:
-      BEAM_SECRET: "${LOCAL_BEAM_SECRET_SPOT}"
+      RUST_LOG: "info"
+      CORS_ORIGIN: http://localhost:5173
```

Comment:
I thinki this is only necessary in local testing and therefore should be part of your local docker-compose.override.yml.

Reply:
Then i'll revert that. Is BEAM_SECRET needed in prod?

Reply:
`BEAM_SECRET` is still needed